### ISSUE TITLE
[BUG] mind model

### ISSUE THREAD

**Issue by JepsonWong** [wziji/deep_ctr#4]
https://github.com/wziji/deep_ctr/blob/0fa3e043300e85be183d219bcdd16aaa3d57e06f/MIND/CapsuleLayer.py#L119

it should be ``weight = tf.nn.softmax(weight, axis=1, name="weight") ``

As you said above, weight's shape is [x,k_max,1], so the dimension softmax would be performed on the second dimension.
